proposal/hsjts0u/README.md

```diff
@@ -0,0 +1,77 @@
+# TermPlot (terminal plotting library)
+
+## Basic Information
+Github repository: [TermPlot](https://github.com/hsjts0u/TermPlot)
+
+## Problem to solve
+Data science and machine learning have become very popular in this day and age, making data visualization an important topic. However, data visualization is quite difficult when there is no access to gui interfaces. There may be times when the user only has access to a terminal (like connecting to a remote server over ssh), which makes data visualization difficult. The user may choose to export the generated graphs, then download the graphs to a local machine with a gui interface, then view the results. These steps are quite redundant, and may be avoided using a terminal graphing library, which is the aim of this project.
+
+## Prospective Users
+Prospective users are those that need data visualization on their terminal for convenience sake, which may include, but not limited to data scientists, machine learning practitionists, and python users.
+
+## System Architecture
+
+### Implementation Detail
+
+#### User Facing Function
+* **scatterplot(height, width, xmin, xmax, ymin, ymax, x, y, graphoptions)**
```

Comment:
If there is only one plotting function, the project will be quite uninteresting.  Could you plan for more plotting functions?  You do not need to implement all of them, but I guess you won't stop with only one.

Reply:
I'll try my best to implement more, hopefully at least one more implementation of a commonly used plot like barplots.